guides/source/action_cable_overview.md

```diff
@@ -671,6 +671,21 @@ To configure the URL, add a call to `action_cable_meta_tag` in your HTML layout
 HEAD. This uses a URL or path typically set via `config.action_cable.url` in the
 environment configuration files.
 
+### Worker Pool Configuration
+
+The worker pool is used to run connection callbacks and channel actions in
+isolation from the server's main thread. Action Cable allows to configure the
```

Comment:
This sentence isn't complete. Can you change "Action Cable allows to configure the amount of simultaneously processed threads in the worker pool" to "Action Cable allows the application to configure the number of simultaneously processed threads in the worker pool". ?